db/schema.rb

```diff
@@ -12,11 +12,6 @@
 
 ActiveRecord::Schema.define(version: 20170127000826) do
 
-  create_table "outcomes", force: :cascade do |t|
```

Comment:
;) Pretty sure you still dont want to remove this model from your database..... 

I would drop your db and rerun your migrations to make sure something funny isnt going on here